backtrader/comminfo.py

```diff
@@ -215,7 +215,7 @@ def getvalue(self, position, price):
         '''Returns the value of a position given a price. For future-like
         objects it is fixed at size * margin'''
         if not self._stocklike:
-            return abs(position.size) * self.get_margin(price)
+            return abs(position.size) * self.get_margin(position.price)
```

Comment:
The method gives the actual *"value"* (or margin requirements) for a given price and not for the price with which the asset was acquired. The usage of the `margin`/`automargin` parameters in the `CommissionInfo` instance, determine if the `price` or the `margin`.